docs/_docs/transfer.md

```diff
@@ -104,20 +103,41 @@ When you’ve finished testing your Transfer-capable trigger and you want to mak
 ### Examples
 
 ```javascript
-const getList = async (z, bundle) => {
-  let attendees = []
-  if (bundle.meta.isBulkRead || bundle.meta.prefill) {
-    // if paginating then use the current page
-    const page = bundle.meta.page ? bundle.meta.page + 1 : 1;
-    attendees = await getAllAttendees(z, bundle, page);
+
+const POLLING_PER_PAGE = {
+  SAMPLE: 3,
+  STANDARD: 30,
+  BULK_READ: 200,
+};
+
+const perform = async (z, bundle) => {
+  let allAthleteActivities = [];
+  if (bundle.meta.isLoadingSample) {
+    // Sample request in the Zap editor, only supports 3 records, so grab 3
+    allAthleteActivities = await getAthleteActivities(z, POLLING_PER_PAGE.SAMPLE, bundle);
+  } else if (bundle.meta.isBulkRead) {
+    // Called from Transfer, so grab the maximum records Strava supports (200)
+    allAthleteActivities = await getAthleteActivities(z, POLLING_PER_PAGE.BULK_READ, bundle);
   } else {
-    // when pulling in attendee samples in the editor we want the most recent
-    attendees = await getRecentAttendees(z, bundle);
+    ) {
+      // Regular running Zap request, use the default (30)
+      allAthleteActivities = await getAthleteActivities(z, POLLING_PER_PAGE.STANDARD, bundle);
+    }
   }
-  return attendees;
+  return allAthleteActivities;
 };
-  ```
-Example of a trigger perform. This API uses the page/offset approach to paging. Here we check `bundle.meta.page` to differentiate handling for a Transfer request vs. a regular Zap handling which just returns the most recent objects.
+
+const getAthleteActivities = async (z, per_page, bundle) => {
+  const response = await z.request({
+    url: `${API_BASE_URL}/athlete/activities`,
+    params: {
+      per_page: per_page,
+      page: bundle.meta.page + 1, // Strava starts paging at 1
+    },
+  });
```

Comment:
Should we return a parsed result here?

```javascript
const getAthleteActivities = async (z, per_page, bundle) => {
  const response = await z.request({
    // ...
  });
  return response.data;
};
```

Reply:
yes we should, where the heck did that go!